inference-engine/thirdparty/clDNN/src/program_helpers.cpp

```diff
@@ -13,24 +13,25 @@
 
 namespace cldnn {
 // helper function for merging the weights/biases buffers on cpu side for depthwise separable convolution optimization
-void program_helpers::merge_buffers(engine_impl& engine,
+void program_helpers::merge_buffers(engine& engine,
                                     program_node& node,
                                     const layout& target_layout,
                                     size_t begin_offset,
                                     size_t end_offset) {
-    memory_impl::ptr data_to_allocate = engine.allocate_memory(target_layout, 0, false);
+    memory::ptr data_to_allocate = engine.allocate_memory(target_layout);
```

Comment:
It looks like memory reset not needed in that case

Reply:
fixed